Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -2919,10 +2928,15 @@ String FrameLoader::userAgent(const URL& url) const
 
     InspectorInstrumentation::applyUserAgentOverride(m_frame, userAgent);
 
+    verifyUserAgent(userAgent);
+    
```

Comment:
Neaten this all up: compute the return value into a local, then call verify, then return it.